src/sentry/receivers/rules.py

```diff
@@ -5,22 +5,26 @@
 from sentry.models import Project, Rule
 
 
+DEFAULT_RULE_LABEL = 'Send a notification for new events'
```

Comment:
I'm using this in getsentry to ignore the first signal created by the default rule.